src/main/scala/uk/gov/hmrc/play/frontend/auth/connectors/AuthConnector.scala

```diff
@@ -31,4 +31,11 @@ trait AuthConnector {
   def currentAuthority(implicit hc: HeaderCarrier): Future[Option[Authority]] = {
     http.GET[Authority](s"$serviceUrl/auth/authority").map(Some.apply) // Option return is legacy of previous http library now baked into this class's api
   }
+  
+  def get[T](uri: String)(implicit hc: HeaderCarrier, reads: HttpReads[T]): Future[T] = {
```

Comment:
as discussed - a convenience method for each URI field of the `AuthContext` (user details, enrolments, IDs)